examples/path-aliases-v1/renderer/onRenderHtml.tsx

```diff
@@ -5,10 +5,11 @@ import ReactDOMServer from 'react-dom/server'
 import React from 'react'
 import { escapeInject, dangerouslySkipEscape } from 'vike/server'
 import { PageLayout } from './PageLayout'
-import type { PageContext } from '#root/types'
+import { ReactComponent } from '#root/types'
+import type { PageContextServer } from 'vike/types'
 
-function onRenderHtml(pageContext: PageContext) {
-  const { Page } = pageContext
+function onRenderHtml(pageContext: PageContextServer) {
+  const Page = pageContext.Page as ReactComponent
```

Comment:
@brillout so I'm forced by TypeScript now to use `PageContextServer` in the signature, which is what we wanted. However it turns out that `PageContextServer['Page']` is of type `unknown`, thus the `as ReactComponent`. This is not elegant, to force the user to do that. Any thoughts?

Reply:
Nevermind, I realize this is what interface merging is for:

```
declare global {
  namespace Vike {
    interface PageContext {
      Page: ReactComponent
    }
  }
}
```

I'll push a new commit

Reply:
Yes, exactly. (It's actually on my to-do list to update the examples and boilerplates to use `Vike.PageContext` instead of `PageContextBuiltIn{Server,Client}`.)